doc/xml/book.xml

```diff
@@ -0,0 +1,211 @@
+                our <literal>image definition tree</literal> here:
+                <link xmlns:xl="http://www.w3.org/1999/xlink" xl:href="https://github.com/SUSE-Enceladus/keg-recipes">Public Cloud Image Definition Tree</link></para>
+    </section>
+    <section xml:id="working-with-keg">
+      <title>Working With Keg</title>
+      <para>Using Keg is done in three steps:</para>
+      <procedure>
+        <step>
+          <para>Fetch or create an <literal>image definition tree</literal></para>
+        </step>
+        <step>
+          <para>Call the <literal>keg</literal> commandline utility to create a KIWI image description</para>
+        </step>
+        <step>
+          <para>Call <literal>kiwi-ng</literal> and create an appliance from the description</para>
+        </step>
+      </procedure>
+      <para>For the above to work, Keg needs to be installed as described in
+                <xref linkend="installation"/>. In addition install KIWI as described here:
+                <link xmlns:xl="http://www.w3.org/1999/xlink" xl:href="https://osinside.github.io/kiwi/installation.html"/></para>
+      <para>If all software components are installed, Keg can be utilized like
+                the following example shows:</para>
+      <screen language="shell-session">$ git clone https://github.com/SUSE-Enceladus/keg-recipes.git
+
+$ keg --recipes-root keg-recipes --dest-dir sles_description \
+      sles/byos/15-sp2
+
+$ sudo kiwi-ng system build --description sles_description \
+      --target-dir kiwi_image</screen>
+    </section>
+  </chapter>
+  <chapter xml:id="installation" xml:base="installation">
+    <title>Installation</title>
+    <note>
+      <para>This document describes how to install Keg. Currently keg is
+                provided from <link xmlns:xl="http://www.w3.org/1999/xlink" xl:href="https://pypi.org/project/kiwi_keg/">PyPi</link> and
+                further install methods for Linux distributions will follow
+                soon.</para>
+    </note>
+    <section xml:id="installation-from-pypi">
+      <title>Installation from PyPI</title>
+      <para>Keg can be obtained from the Python Package Index (PyPi) via Python’s
+                package manager pip:</para>
+      <screen language="shell-session">$ pip install kiwi_keg</screen>
+    </section>
+  </chapter>
+  <chapter xml:id="command-line" xml:base="commands">
+    <title>Command Line</title>
+    <section xml:id="keg" xml:base="commands/keg">
+      <title>keg</title>
+      <section xml:id="keg-synopsis">
+        <title>SYNOPSIS</title>
+        <screen language="bash">keg (-r RECIPES_ROOT|--recipes-root=RECIPES_ROOT)
+    [-a ADD_DATA_ROOT] ... [-d DEST_DIR] [-fv]
+    SOURCE
+keg -h | --help
+keg -h | --help</screen>
+      </section>
+      <section xml:id="description">
+        <title>DESCRIPTION</title>
+        <para>Keg is a tool which helps to create and manage image descriptions suitable
```

Comment:
repeating paragraph from `conceptual overview` ? 
IMO, at this point, what `keg` is has been already explained, maybe mention that it can be used from command line or as a CLI tool

thoughts ?

Reply:
The reason why you see this twice is that one source is for the manual page and the other is for the documentation in the conceptual overview. When reading the docs as a single document I agree with you the command info and the conceptual overview contains the same information. But users on the commandline that runs ```man keg``` expect the man page to say at least some descriptive words. If we change this and write "keg can be used on the commandline as a CLI tool" in a man page, it would be considered a joke probably :)

So sphinx renders the source for different targets and I like if man pages also appears in the html documentation. That's why there is an overlap for the description information.

Reply:
Hope this makes sense, please unresolve if not